crates/sui-cluster-test/src/config.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use clap::*;
+
+#[derive(Parser, Clone, ArgEnum)]
+pub enum Env {
+    Prod,
+    Staging,
+    CustomRemote,
+    NewLocal,
+}
```

Comment:
This shouldn't be apart of the opts passed to the factory because what do you do if you try to set "pod" when then factory you have is Local? instead this should be something that is at a slightly higher level which selects the type of factory to use.